packages/cashier/src/pages/withdrawal/withdrawal-crypto-form/withdrawal-crypto-form.scss

```diff
@@ -44,4 +44,8 @@
             }
         }
     }
+
+    & .withdraw__input {
+        margin-bottom: 3.2rem !important;
```

Comment:
Shouldn't we be avoiding hardcoding styles using `!important`? Couldn't this issue be solved by removing 
```
@include mobile {
    margin-top: 0;
}
```
in __percentage-container above (unless if that property is needed)?

Reply:
I tried the above but it does not work  since the `!important` is also used for the same class in different places to override the behavior in those places:
<img height="450" alt="image" src="https://github.com/binary-com/deriv-app/assets/125039206/bbfc9e28-17e9-4ce8-a8f4-94c0c3d4ddcf">
![image](https://github.com/binary-com/deriv-app/assets/125039206/a0075d93-2564-4719-a399-5c145800cd12)
![image](https://github.com/binary-com/deriv-app/assets/125039206/0b7d5c8f-0e72-473c-b43e-fb9bd8cee9e0)

Reply:
I tried commenting it out and it seems to work 🤔 but if it's situational then the implementation should be fine as it's already being used

<img width="40%" alt="Screenshot 2023-11-30 at 6 28 23 PM" src="https://github.com/binary-com/deriv-app/assets/142860499/a88818ec-98c1-407a-a2a6-6bd5bb12eef9">

<img width="40%" alt="Screenshot 2023-11-30 at 6 28 23 PM" src="https://github.com/binary-com/deriv-app/assets/142860499/482718a2-0316-4931-960b-b29cba9daefc">

Reply:
My bad, applied it in the wrong place 😅.
This works and have applied the change.